repose-aggregator/core/repose-core/src/main/java/org/openrepose/nodeservice/httpcomponent/HttpComponentRequestProcessor.java

```diff
@@ -20,6 +20,7 @@
 package org.openrepose.nodeservice.httpcomponent;
 
 import org.apache.commons.io.IOUtils;
+import org.apache.commons.lang.ArrayUtils;
```

Comment:
You probably want to import from the `lang3` package. This imports a previous version which is not controlled by our build. I suppose it must be transitive, but since the artifact ID changed to `commons-lang3`, we end up with multiple versions of this library. Still the classes themselves have different packages, so we can specify that we want the newer one.

Reply:
Will do.